src/Forms/DiffTransformation.php

```diff
@@ -110,7 +110,7 @@ public function transform(FormField $field)
                 Diff::compareHTML($field->Value(), isset($this->data[$name]) ? $this->data[$name] : '')
             );
 
-            $diffField->addExtraClass($field->extraClass());
+            $diffField->addExtraClass($field->extraClass() . "history-viewer__version-detail-diff");
```

Comment:
Please add a space at the start of the quoted string, or add it with a separate call to `addExtraClass`